runtime/hydradx/src/tests.rs

```diff
@@ -329,9 +329,15 @@ mod xcm_fee_payment_api_tests {
 				},
 			]);
 
-			assert_eq!(
-				Runtime::query_xcm_weight(VersionedXcm::from(xcm_message.clone())),
-				Ok(Weight::from_parts(200000000, 0))
+			let queried_weight = Runtime::query_xcm_weight(VersionedXcm::from(xcm_message.clone())).unwrap();
+			let ref_time = queried_weight.ref_time();
+			assert!(
```

Comment:
Let's not use any range. Assert that it returns `Ok` with some value should be enough.

Reply:
What is the issue with using range?

The problem with specific assertion is that that it will fail every time we rebenchmark. By having a range, it will fail only if the weight significantly changes, which is good, I mean, we wanna be aware of that.